### ISSUE TITLE
Inquiry About Location of Retrieval Databases in Testing Phase

### ISSUE THREAD

**Issue by Jack-ZC8** [richard-peng-xia/MMed-RAG#25]
Hello,

I would like to ask if the retrieval databases being used in the testing phase are located in the following three files? I haven't found the retrieval report databases in any other locations.

- `data/training/retriever/ophthalmology/harvard_train_7000.json`
- `data/training/retriever/pathology/pathology_train.json`
- `data/training/retriever/radiology/radiology_train.json`

Thank you for your assistance!

**Comment by Jack-ZC8** [comment]
Additionally, it seems that the retrieval database used in the testing phase is the same as the data used for training the retriever. If there are any misunderstandings, please feel free to correct me.

**Comment by richard-peng-xia** [comment]
Hi @Jack-ZC8,

Currently, we only provide data for retrieval model training. Because the data in the validation set for the retriever does not matter, you only need to select a part of it from the same source dataset. Note that it cannot be repeated with the training set.
